3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -111,11 +111,35 @@
     print(f"Added single cell count as metadata to {pathlib.Path(dest_path).name}!")
 
 
-# ### Check if converted data looks correct
+# ## Update the files to remove NA rows added as artifacts of CytoTable
```

Comment:
You might clarify that you are actively rewriting the files - this seems like an important step for reproducibility that might be worth highlighting, perhaps in a README somewhere? Specifically, I can see someone running CytoTable but forgetting this step. Up to you!

Reply:
A note has been added into the README for the module. Thanks for this idea!